src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-list/iscsi-target-list.component.html

```diff
@@ -0,0 +1,38 @@
+<cd-iscsi-tabs></cd-iscsi-tabs>
+
+<cd-info-panel *ngIf="available === false"
+               title="iSCSI Targets not available"
+               i18n-title>
+  <ng-container i18n>Please consult the <a href="{{docsUrl}}"
+       target="_blank">documentation</a>
+    on how to configure and enable the iSCSI Targets management functionality.</ng-container>
+
+  <ng-container *ngIf="status">
+    <br>
+    <span i18n>Available information:</span>
+    <pre>{{ status }}</pre>
```

Comment:
The pre-formated text does not look nice. I don't have an alternative idea yet, but i think this needs to be polished a little bit.
![auswahl_002](https://user-images.githubusercontent.com/1897962/52057564-9b5d0500-2565-11e9-9132-5d0d6105f0ad.png)